runtime/doc/vim_diff.txt

```diff
@@ -30,6 +30,7 @@ these differences.
 
 - 'backspace' defaults to "indent,eol,start"
 - 'encoding' defaults to "utf-8"
+- 'formatoptions'  includes "j" (see |fo-table|)
```

Comment:
Sure, [I don't have strong feelings one way or the other](https://www.youtube.com/watch?v=CxK_nA2iVXw)